src/instrumentation/do-storage.ts

```diff
@@ -41,25 +63,116 @@ const StorageAttributes: Record<string | symbol, ExtraAttributeFn> = {
 			}
 		}
 		if (argArray.length > 1) {
-			Object.assign(attrs, argArray[1])
+			const options = argArray[1] as DurableObjectPutOptions
+			if ('allowConcurrency' in options) {
+				attrs['db.cf.do.allow_concurrency'] = options.allowConcurrency
+			}
+			if ('noCache' in options) {
+				attrs['db.cf.do.no_cache'] = options.noCache
+			}
 		}
 		return attrs
 	},
 	list(argArray, result: Map<string, unknown>) {
-		// list may be called with no arguments
 		const attrs: Attributes = {
 			'db.cf.do.number_of_results': result.size,
+		} as Record<string, string | number | boolean | undefined>
+		if (argArray.length > 0) {
+			const options = argArray[0] as DurableObjectPutOptions
+			if ('allowConcurrency' in options) {
+				attrs['db.cf.do.allow_concurrency'] = options.allowConcurrency
+			}
+			if ('noCache' in options) {
+				attrs['db.cf.do.no_cache'] = options.noCache
+			}
+			if ('start' in options) {
+				attrs['db.cf.do.start'] = options.noCache
```

Comment:
Why are we assigning `options.noCache` to the rest of these? (guessing this is a copy/paste issue)